app/views/ticket_requests/_form.html.haml

```diff
@@ -87,62 +87,65 @@
-                            = f.label :role_contributor, class: 'radio inline' do
-                              = f.radio_button :role, TicketRequest::ROLE_CONTRIBUTOR,
-                                data: { 'max-tickets' => TicketRequest::TICKET_LIMITS[TicketRequest::ROLE_CONTRIBUTOR] }
-                              = TicketRequest::ROLES[TicketRequest::ROLE_CONTRIBUTOR]
-                            = f.label :role_coordinator, class: 'radio inline' do
-                              = f.radio_button :role, TicketRequest::ROLE_COORDINATOR,
-                                data: { 'max-tickets' => TicketRequest::TICKET_LIMITS[TicketRequest::ROLE_COORDINATOR] }
-                              = TicketRequest::ROLES[TicketRequest::ROLE_COORDINATOR]
-                            = f.label :role_uber_coordinator, class: 'radio inline' do
-                              = f.radio_button :role, TicketRequest::ROLE_UBER_COORDINATOR,
-                                data: { 'max-tickets' => TicketRequest::TICKET_LIMITS[TicketRequest::ROLE_UBER_COORDINATOR] }
-                              = TicketRequest::ROLES[TicketRequest::ROLE_UBER_COORDINATOR]
-                            = f.label :role_other, class: 'radio inline' do
-                              = f.radio_button :role, TicketRequest::ROLE_OTHER,
-                                data: { 'max-tickets' => TicketRequest::TICKET_LIMITS[TicketRequest::ROLE_OTHER] }
-                              = TicketRequest::ROLES[TicketRequest::ROLE_OTHER]
-
-                  .col-lg-6.col-xl-6.col-md-12.col-sm-12.align-content-md-center
-                    %p.muted.role-explanation{ class: TicketRequest::ROLE_VOLUNTEER }
-                      You are working one or more shifts at the event
-
-                    %p.muted.role-explanation.hidden{ class: TicketRequest::ROLE_CONTRIBUTOR }
-                      Working before and during or during and after; actively involved in
-                      planning the campout e.g. a coordinator assistant or apprentice. Bringing
-                      gear, driving trucks, art committee, etc.
-
-                    %p.muted.role-explanation.hidden{ class: TicketRequest::ROLE_COORDINATOR }
-                      Listed on the Coordinator Sheet as someone who is leading a major area of
-                      camp out planning; working before, during, and after the event including
-                      things like resource coordination, significant gear prep, pre/post-site
-                      visits, etc.
-
-                    %p.muted.role-explanation.hidden{ class: TicketRequest::ROLE_OTHER }
-                      Art Grantee or DJ
-
-                    = f.text_area :role_explanation,
-                      placeholder: 'Briefly describe your role',
-                      rows: 4, width: 400, maxlength: 200, required: false, style: 'min-width: 100% !important; font-size: 11pt;'
+              - if @event.require_role
```

Comment:
This chunk is large enough that I'd move it into an `_event_role.html.haml` partial in the same folder.

Then right after `if` add render partial: :event_role`